src/fty_email_server.cc

```diff
@@ -1036,8 +1036,8 @@ test9 (bool verbose, const char *endpoint)
 
     // this alert is NOT supposed to be in the file,
     // as action EMAIL is NOT specified
-    msg = fty_proto_encode_alert (NULL, "SOME_RULE1", "SOME_ASSET", \
-                                  "ACTIVE","CRITICAL","ASDFKLHJH", 123456, "SMS", 600);
+    msg = fty_proto_encode_alert (NULL, time (NULL), 600, "SOME_RULE1", "SOME_ASSET", \
```

Comment:
Just in case, shouldn't references to `time()` be `std::time()` or `::time()`, to avoid conflicts with C++ overrides, as was done in original code (below)? Just recently fixed similar mixup with `isnan()`, so just asking ;)